lib/rake/task_manager.rb

```diff
@@ -301,6 +321,33 @@ def get_description(task)
       desc
     end
 
+    # Return a value representing the "cost" of transforming str1 into str2
+    def levenshtein_distance(str1, str2)
```

Comment:
It is. We can just `include Gem::Text`and call `#levenshtein_distance`. I think you already know it had a bug, but maybe we shouldn't worry about it?